src/main/scala/nl.knaw.dans.easy.properties/app/graphql/types/MutationType.scala

```diff
@@ -51,7 +51,16 @@ trait MutationType {
   case class SetSpringfieldPayload(clientMutationId: Option[String], objectId: String) extends Mutation
   case class SetContentTypePayload(clientMutationId: Option[String], objectId: String) extends Mutation
   case class RegisterDepositPayload(clientMutationId: Option[String], depositId: DepositId) extends Mutation
+  case class DeleteDepositsPayload(clientMutationId: Option[String], depositIds: Seq[DepositId]) extends Mutation
 
+  private val depositIdListInputField: InputField[Seq[DepositId]] = InputField(
+    name = "depositId",
+    description = Some("The identifiers of the deposits to delete."),
```

Comment:
This `InputField` may be reused in the future, so I would not want to state the 'delete' part in the description here.

```suggestion
    description = Some("A list of deposit identifiers."),
```